posthog/api/capture.py

```diff
@@ -230,12 +236,14 @@ def drop_events_over_quota(token: str, events: List[Any]) -> List[Any]:
 
     for event in events:
         if event.get("event") in SESSION_RECORDING_EVENT_NAMES:
+            EVENTS_RECEIVED_COUNTER.labels(resource_type="recordings").inc()
             if token in limited_tokens_recordings:
                 EVENTS_DROPPED_OVER_QUOTA_COUNTER.labels(resource_type="recordings", token=token).inc()
                 if settings.QUOTA_LIMITING_ENABLED:
                     continue
 
         elif token in limited_tokens_events:
+            EVENTS_RECEIVED_COUNTER.labels(resource_type="events").inc()
```

Comment:
this isn't super clear to me ... even if this includes all teams now it might not in the future, I's suggest similar to recordings handling:
```
else:
  EVENTS_RECEIVED_COUNTER.labels(resource_type="events").inc()
  if token in limited_tokens_events:
    ...
```